libcodechecker/libhandlers/server.py

```diff
@@ -755,8 +755,8 @@ def server_init_start(args):
 
     # Create the main database link from the arguments passed over the
     # command line.
-    default_product_path = os.path.join(args.config_directory,
-                                        'Default.sqlite')
+    cfg_dir = os.path.abspath(args.config_directory)
```

Comment:
I think this is not the best way to handle this problem. This way every relative path will be converted to absolute path. What if the user would like to move his config directory to a different directory?

Reply:
Right now we write every product which is added later with an absolute path to the config db.
Only the default was written as a relative path.

Reply:
You need to be very careful with `abspath` if it is a relative path, as then it will relative to the working directory **at the point of execution**.